packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/imageEditors/Rotator.ts

```diff
@@ -85,7 +85,7 @@ export function getRotateHTML({
                 {
                     tag: 'div',
                     className: ImageEditElementClass.RotateHandle,
-                    style: `position:absolute;background-color:${rotateHandleBackColor};border:solid 1px ${borderColor};border-radius:50%;width:${ROTATE_SIZE}px;height:${ROTATE_SIZE}px;left:-${handleLeft}px;cursor:move`,
+                    style: `position:absolute;background-color:${rotateHandleBackColor};border:solid 1px ${borderColor};border-radius:50%;width:${ROTATE_SIZE}px;height:${ROTATE_SIZE}px;left:-${handleLeft}px;cursor:move;z-index:2`,
```

Comment:
Why need z-index here?

Reply:
So the handle do not be overlaid by the handle.

Reply:
Can we solve this issue by change the rendering order? If possible, avoid using zIndex